### ISSUE TITLE
Unable to identify "Hilliary Clinton" in input

### ISSUE THREAD

**Issue by yongzhihuang** [nlp-compromise/compromise#284]
Hi! 

When using Hillary Clinton as an input to this library:

```
The second presidential debate contained a slew of troubling moments for both sides. Among the most disturbing, existentially speaking, was a comment that Donald Trump made about Hillary Clinton that if realized would threaten the balance of power within the American democracy. 
```

it produces the following result:

```
Person {
  whitespace: { preceding: '', trailing: ' ' },
  text: 'Clinton',
  normal: 'clinton',
  expansion: null,
  reasoning: [ 'capital_signal', 'lexicon_pass' ],
  pos: { Noun: true, Person: true, MalePerson: true },
  tag: 'MalePerson',
  honourific: null,
  firstName: 'Clinton',
  middleName: '',
  lastName: null }
```

Do you know what the possible issue might be?

**Comment by davidbuhler** [comment]
firstName: 'Clinton',

**Comment by yongzhihuang** [comment]
@davidbuhler That's a wrong interpretation it seems, i would like First name to be Hillary.

**Comment by spencermountain** [comment]
hi Yongzhi, ouch, this one hurts a bit.
yeah, it looks like `Clinton` is being recognized as another first name, then it's refusing to lump them together.
this should do the trick:
```javascript
lexicon = nlp.lexicon()
lexicon["apple"] = "Person" // augment it
let s = nlp.sentence('apples are nice', {lexicon:lexicon}); 
console.log(s.tags()); 
```
but really, it's a bug i should fix.
will leave this open,
thanks!

**Comment by yongzhihuang** [comment]
@spencermountain Great, thank you very much for looking into it.

**Comment by spencermountain** [comment]
hey Yongzhi, this is fixed now in `7.0.9`
```javascript
var str='The second presidential debate contained a slew of troubling moments for both sides. Among the most disturbing, existentially speaking, was a comment that Donald Trump made about Hillary Clinton that if realized would threaten the balance of power within the American democracy.'
console.log(nlp(str)).people().data());
//[donald trump, hilary clinton]
```
cheers!
